src/CoreWCF.BuildTools/src/Extensions.cs

```diff
@@ -123,4 +139,23 @@ public static string ToSafeCSharpString(this TypedConstant typedConstant)
 
         }
     }
+
+    internal static class StringExtensions
+    {
+        public static string? ToMessagePropertyVariableName(this string? propertyName)
+        {
+            if (propertyName is null)
+            {
+                return null;
+            }
+
+            var parts = propertyName.Split('.');
+            return parts.Last().ToCamelCase();
+        }
```

Comment:
Why are we forcing this to be camel case? Someone could put in their own property with any casing they like.

Reply:
Nevermind, I see this is for generating nice looking parameter names in the generated code.